src/firmware/linux/guest/types.rs

```diff
@@ -347,6 +451,29 @@ bitfield! {
     pub single_socket_required, _: 20, 20;
 }
 
+impl Display for SnpGuestPolicy {
```

Comment:
From clippy:
`using self as Display in impl Display will cause infinite recursion`